WordPress/gutenberg
# Fix last.fm svg icon alignment

## Description

Fix #24272.

Fix the last.fm icon alignment in social icon block. I use [SvgPathEditor](https://yqnn.github.io/svg-path-editor/) to edit the `<path>` of the icon to translate it 1.5px x and y.

## How has this been tested?



1. Add a new social icon block.
2. Add a last.fm icon along with other social icons.
3. The icon should have the same alignment and sizes as the others.

## Screenshots 

**Default**
![Screen Shot 2020-07-30 at 10 51 27 AM](https://user-images.githubusercontent.com/7753001/88875774-79286e00-d254-11ea-9bf5-4665ee2ff781.png)

**Logos Only**
![Screen Shot 2020-07-30 at 10 52 03 AM](https://user-images.githubusercontent.com/7753001/88875891-af65ed80-d254-11ea-8426-1189307bc34e.png)

**Pill Shape**
![Screen Shot 2020-07-30 at 10 52 18 AM](https://user-images.githubusercontent.com/7753001/88875916-bbea4600-d254-11ea-90dc-46b4ce87e2e3.png)

## Types of changes




Bug fix

## Checklist:
